pkgs/applications/audio/rofi-mpd/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, python37Packages, fetchFromGitHub, pkgs }:
```

stdenv should be avoided when possible, pkgs isn't used
```suggestion
{ lib, python37Packages, fetchFromGitHub }:
```